Source/diablo.cpp

```diff
@@ -1777,6 +1773,12 @@ void InitKeymapActions()
 	    N_("Pauses the game."),
 	    'P',
 	    diablo_pause_game);
+	sgOptions.Keymapper.AddAction(
+	    "Pause Game (Alternate)",
+	    N_("Pause Game"),

```

Comment:
Should we also add (Alternate) to the displayed text in the settings menu?
Else we would have two entries with "Pause Game" in the settings menu. That could be confusing.

Reply:
Maybe, yeah. The only confusion I've seen regarding this in padmapper was when someone was trying to explain their bindings to me on Discord. I don't know if the extra text would have helped, but it couldn't hurt, right?

Reply:
I thought about some issues like "duplicate entries in keymapper settingsmenu" or something like this. 😁
And as you said some extra text shouldn't hurt.